canvas_account_admin_tools/templates/canvas_account_admin_tools/dashboard_account.html

```diff
@@ -44,28 +44,34 @@ <h4 class="list-group-item-heading">LTI Tools Usage Report</h4>
 
       {% if courses_in_this_account %}
         <div class="list-group col-md-4">
-          <a type="button" class="btn btn-primary list-group-item active" href="{{ courses_in_this_account }}" class="list-group-item active" target="_parent">
+          <a type="button" class="btn btn-primary list-group-item active" href="{{ courses_in_this_account }}" target="_parent">
               <h4 class="list-group-item-heading">Courses in this Account</h4>
               <p class="list-group-item-text">See information about the courses in your account combined in one view</p>
           </a>
         </div>
       {% endif %}
 
         <div class="list-group col-md-4">
-          <a type="button" class="btn btn-primary list-group-item active" href="{% url 'course_info:index' %}" class="list-group-item active">
+          <a type="button" class="btn btn-primary list-group-item active" href="{% url 'course_info:index' %}">
               <h4 class="list-group-item-heading">Find Course Info</h4>
               <p class="list-group-item-text">Search for course information <br>&nbsp;</p>
           </a>
         </div>
 
       {% if cross_listing_allowed %}
         <div id="cross-listing-button" class="list-group col-md-4">
-          <a type="button" class="btn btn-primary list-group-item active" href="{% url 'cross_list_courses:index' %}" class="list-group-item active">
+          <a type="button" class="btn btn-primary list-group-item active" href="{% url 'cross_list_courses:index' %}">
               <h4 class="list-group-item-heading">Cross-list Courses</h4>
               <p class="list-group-item-text">Set up and modify cross-listings <br>&nbsp;</p>
           </a>
         </div>
       {% endif %}
 
+        <div class="list-group col-md-4">
```

Comment:
Need an `{% if people_tool_allowed %}` and the corresponding code in `views.py` to check the the new `people_tool` `LtiPermission`.